### DISCUSSION TITLE
Attach to multiple servers

### DISCUSSION THREAD

**Discussion by N4tus** [zellij-org/zellij#2470] [Ideas]
IT would be really cool if it were possible to attach to multiple servers. Merging the tabs from each, such that some tabs are backed by one server and others by the other server.  
The main usecase for this is to connect to a zellij-session on another computer over ssh.

**Comment by Lockszmith-GH** [comment]
Zellij is about local sessions, you can run zellij on the remote device, or open a new connection in a new zellij pane.

I do this with my machines, where my remote machines have a zellij config with a different lock-key-bind (Ctrl-B) and my local machine using the default lock-key-bind (Ctrl-G) (also use different themes, to visually distinguish quickly where I'm working)

**Comment by EvanEdwards** [comment]
I agree with @Lockszmith-GH -- but I also think it would be advantageous to have the ability to have *sessions* be able to be connected to other systems so you can switch between them, detach, reattach, etc.  I have not dug in enough (I'm just now flipping to zellij as a primary system from about 30 years of Screen and TMUX), but the server/client system for reattaching sessions seems new, and plausibly could support this: a session with a host that can be connected over an SSH connection.

"Plausible" with a grain of salt the size of my ignorance about the implementation, which is extremely large.   I tripped across this while looking to see if it had been proposed or was being worked on.
